x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/50_token_auth.yml

```diff
@@ -1,180 +0,0 @@
----
-"Get the indexed token and use if to authenticate":
-  - skip:
-      features: headers
-
-  - do:
-      cluster.health:
-          wait_for_status: yellow
-
-  - do:
-      get:
-        index: token_index
-        id:    "6"
-
-  - match: { _index:   token_index }
-  - match: { _id:      "6"      }
-  - is_true: _source.token
-  - set: { _source.token : token }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      security.authenticate: {}
-
-  - match: { username: "token_user" }
-  - match: { roles.0:  "superuser" }
-  - match: { full_name: "Token User" }
-
-  # call three times because the client rotates the nodes
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
----
-"Get the indexed refreshed access token and use if to authenticate":
-  - skip:
-      features: headers
-
-  - do:
-      get:
-        index: token_index
-        id:    "7"
-
-  - match: { _index:   token_index }
-  - match: { _id:      "7"      }
-  - is_true: _source.token
-  - set: { _source.token : token }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      security.authenticate: {}
-
-  - match: { username: "token_user" }
-  - match: { roles.0:  "superuser" }
-  - match: { full_name: "Token User" }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
-  - do:
-      headers:
-        Authorization: Bearer ${token}
-      search:
-        rest_total_hits_as_int: true
-        index: token_index
-
-  - match: { hits.total: 8 }
-
----
-"Get the indexed refresh token and use it to get another access token and authenticate":
```

Comment:
Covered by: https://github.com/elastic/elasticsearch/blob/main/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/TokenBackwardsCompatibilityIT.java#L169

In the above Java REST test, we "verify new nodes can refresh tokens created by old nodes and vice versa" so the scenario of using an old token on a new node is covered